Apps/W1/HybridGP/app/src/Migration/Support/CheckBooks/GPCheckbookMSTR.table.al

```diff
@@ -178,94 +178,4 @@ table 40099 "GP Checkbook MSTR"
             Clustered = true;
         }
     }
-
-    procedure MoveStagingData()
```

Comment:
We cannot move public methods - it is a breaking change. I love the refactoring done, but public method should stay and there should be a one line reference to the new method.

Reply:
would the one line reference allow this method to still perform the migration or is it just a comment noting where the code was moved to?